tests/cucumber/e2e/stepDefinations/addmovieContext.js

```diff
@@ -0,0 +1,39 @@
+const { Given, When, Then } = require("@cucumber/cucumber");
+// import expect for assertion
+const { expect } = require("@playwright/test");
+const addBtn = `//button[@id="titleaddBtn"]`;
+const form = `//div[@id="myDropdown"]`;
+const searchRes = `//div[@id="searchTitle"]//div/a`;
+const dropdown = `//div[@id="myDropdown"]`;
+const headderText=`//div[@class="title"]/h1`
+Given(
+    "user has navigated to the details page of movie {string}",
+    async function (string) {
```

Comment:
```suggestion
    async function (movieName) {
```
Use meaningful name for variables